x/spec/keeper/spec_test.go

```diff
@@ -770,8 +770,8 @@ func TestApiCollectionsExpandAndInheritance(t *testing.T) {
 func TestCookbookSpecs(t *testing.T) {
 	ts := newTester(t)
 
-	getToTopMostPath := "../../../"
-	proposalFile := "./cookbook/specs/spec_add_ibc.json,./cookbook/specs/spec_add_cosmoswasm.json,./cookbook/specs/spec_add_cosmossdk.json,./cookbook/specs/spec_add_cosmossdk_full.json,./cookbook/specs/spec_add_ethereum.json,./cookbook/specs/spec_add_cosmoshub.json,./cookbook/specs/spec_add_lava.json,./cookbook/specs/spec_add_osmosis.json,./cookbook/specs/spec_add_fantom.json,./cookbook/specs/spec_add_celo.json,./cookbook/specs/spec_add_optimism.json,./cookbook/specs/spec_add_arbitrum.json,./cookbook/specs/spec_add_starknet.json,./cookbook/specs/spec_add_aptos.json,./cookbook/specs/spec_add_juno.json,./cookbook/specs/spec_add_polygon.json,./cookbook/specs/spec_add_evmos.json,./cookbook/specs/spec_add_base.json,./cookbook/specs/spec_add_canto.json,./cookbook/specs/spec_add_sui.json,./cookbook/specs/spec_add_solana.json,./cookbook/specs/spec_add_bsc.json,./cookbook/specs/spec_add_axelar.json,./cookbook/specs/spec_add_avalanche.json,./cookbook/specs/spec_add_fvm.json"
+	getToTopMostPath := "../../.././cookbook/specs/"
+	proposalFile := "spec_add_ibc.json,spec_add_cosmoswasm.json,spec_add_cosmossdk.json,spec_add_cosmossdk_full.json,spec_add_ethereum.json,spec_add_cosmoshub.json,spec_add_lava.json,spec_add_osmosis.json,spec_add_fantom.json,spec_add_celo.json,spec_add_optimism.json,spec_add_arbitrum.json,spec_add_starknet.json,spec_add_aptos.json,spec_add_juno.json,spec_add_polygon.json,spec_add_evmos.json,spec_add_base.json,spec_add_canto.json,spec_add_sui.json,spec_add_solana.json,spec_add_bsc.json,spec_add_axelar.json,spec_add_avalanche.json,spec_add_fvm.json,spec_add_mantle.json"
```

Comment:
read all the files in the folder instead of maintaining a list

Reply:
you can rename them by adding a numeric prefix, e.g. `01_spec_add_ibc.json` etc to enforce order
(similar to how `run-parts(8)` works)

Reply:
possible but ugly